<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private 'security.firewall.map.context.login' shared service.

include_once $this->targetDirs[3].'/vendor/symfony/security-bundle/Security/FirewallContext.php';

return $this->privates['security.firewall.map.context.login'] = new \Symfony\Bundle\SecurityBundle\Security\FirewallContext(new RewindableGenerator(function () {
    yield 0 => ($this->privates['security.channel_listener'] ?? $this->load('getSecurity_ChannelListenerService.php'));
    yield 1 => ($this->privates['security.authentication.listener.json.login'] ?? $this->load('getSecurity_Authentication_Listener_Json_LoginService.php'));
    yield 2 => ($this->privates['security.authentication.listener.anonymous.login'] ?? $this->load('getSecurity_Authentication_Listener_Anonymous_LoginService.php'));
    yield 3 => ($this->privates['security.access_listener'] ?? $this->load('getSecurity_AccessListenerService.php'));
}, 4), ($this->privates['security.exception_listener.login'] ?? $this->load('getSecurity_ExceptionListener_LoginService.php')), NULL, ($this->privates['security.firewall.map.config.login'] ?? $this->load('getSecurity_Firewall_Map_Config_LoginService.php')));
